app/templates/frameworks/services.html

```diff
@@ -65,91 +64,115 @@ <h2 class="summary-item-heading">{{ framework.name }} is closed for applications
-    field_headings=[
-        "Service name",
-        "Progress",
-        "Make a copy"
-    ],
-    field_headings_visible=False
-  ) %}
-    {% call summary.row() %}
-      {{ summary.service_link(
-          draft.serviceName,
-          url_for(".view_service_submission", framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id)
-      ) }}
-      {% if framework.status == 'open' %}
-        {% call summary.field(action=True) %}
-          <form method="post" action="{{ url_for('.copy_draft_service', framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id) }}">
-            <input type="hidden" name="csrf_token" value="{{ csrf_token_value or csrf_token() }}" />
-            {{ govukButton({
-              "text": "Make a copy",
-              "classes": "govuk-!-margin-0",
-            }) }}
-          </form>
-        {% endcall %}
-      {% else %}
-        {{ summary.text() }}
-      {% endif %}
-    {% endcall %}
-  {% endcall %}
 
-  <hr class="govuk-section-break govuk-section-break--m">
+  {% if not complete_drafts %}
+  {{ "You haven’t marked any services as complete yet." if framework.status == 'open' else "You didn’t mark any services as complete."  }}
+  {% endif %}
+  {% set ns = namespace(completed_rows = []) %}
+   {% if framework.status == 'open' %}
+    {% for complete_draft in complete_drafts %}
+    {% set service_link %}
+    <a class="govuk-link" href="{{ url_for('.view_service_submission', framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id) }}">{{ complete_draft.serviceName }}</a>
+    {% endset %}
+    {% set make_copy_button %}
+    <form method="post" action="{{ url_for('.copy_draft_service', framework_slug=framework.slug, lot_slug=complete_draft.lot, service_id=complete_draft.id) }}">
+        <input type="hidden" name="csrf_token" value="{{ csrf_token_value or csrf_token() }}" />
+        {{ govukButton({
+          "text": "Make a copy",
+          "classes": "govuk-!-margin-0",
+        }) }}
+    </form>
+    {% endset %}
+        {% set row = ns.completed_rows.append(
+        [
+          {"html": service_link},
+          {"html": make_copy_button}
+        ]
+    ) %}
+        {{ govukTable({
+      'head': [
+        {'text': "Service name"},
+        {'text': "Make a copy"}
+      ],
+      'rows': ns.completed_rows,
+      "classes": "dm-table govuk-!-margin-bottom-6",
+    }) }}
+    {% endfor %}
+    {% elif framework.status == 'pending' or 'stanstill' %}
+    <ul class="govuk-list">
+        {% for complete_draft in complete_drafts  %}
+            <li>
+                <a class="govuk-link" href="{{ url_for('.view_service_submission', framework_slug=framework.slug, lot_slug=complete_draft.lot, service_id=draft.id) }}">{{ complete_draft.serviceName }}</a>
+            </li>
+        {% endfor %}
+      </ul>
```

Comment:
I have replaced the table in the pending/standstill state with a simple list as there is only one row. Not sure if we want to replicate the previous style?

## Before
![Screenshot 2021-06-17 at 15 48 38](https://user-images.githubusercontent.com/1764158/122420394-90af3900-cf83-11eb-8549-bda0757503bf.png)


## After
![Screenshot 2021-06-17 at 15 47 37](https://user-images.githubusercontent.com/1764158/122420208-68273f00-cf83-11eb-98e5-319b64ba03da.png)

Reply:
No, I think this makes sense.

Anywhere we list services (in tables or otherwise), the page looks ridiculous if the user has tons of services, but that's a problem for another day.